docs/installation/upgrading.rst

```diff
@@ -47,4 +47,14 @@ Upgrading from KSQL 5.0.0 and below to KSQL 5.1
 
     * The KSQL engine metrics are now prefixed with the ``ksql.service.id``. If you have been using any metric monitoring
       tool you need to update your metric names.
-      For instance, assuming ``ksql.service.id`` is set to ``default``, ``messages-produced-per-sec`` will be changed to ``_confluent-ksql-default_messages-consumed-per-sec``.
\ No newline at end of file
+      For instance, assuming ``ksql.service.id`` is set to ``default``, ``messages-produced-per-sec`` will be changed to ``_confluent-ksql-default_messages-consumed-per-sec``.
+
+* Configuration:
+
+    * When upgrading your headless (non-interactive) mode application, you must include the following configurations in your properties file. Note that this is not required for interactive mode KSQL:
```

Comment:
@rodesai : Shouldn't we also update this section so that users understand that there is a choice to make:  either (1) upgrade your queries that use SUBSTRING, or (2) set the legacy config option to true.  And we do recommend users to go with (1) when possible.

Reply:
Our wording in the other changed section is better than this (initial/original) text from an earlier state of this PR.